### PR TITLE
Added the :alt: attribute 

### PR DESCRIPTION
**Pull request by Dule-martins**
##### SUMMARY

Added the :alt: attribute to display alt text for all image

related #14505 

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - Docs

##### AWX VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: docs/docsite/rst/userguide/inventories.rst [hunk 1]
```diff
@@ -246,6 +256,7 @@ The hosts inside the input inventory will fit one condition, the other condition
 neither, or both. This results in four hosts total for demonstration purposes.
 
 .. image:: ../common/images/inventories-constructed-inventory-venn.png
+   :alt: Inventories constructed invontory venn
```
Comment (gundalow) [inline docs/docsite/rst/userguide/inventories.rst]:
```suggestion
   :alt: Inventories constructed inventory Venn diagram
```

File: docs/docsite/rst/userguide/inventories.rst [hunk 2]
```diff
@@ -459,6 +470,7 @@ Follow the procedure described in the subsequent section, :ref:`ug_inventories_a
 Example of a constructed inventory details view:
 
 .. image:: ../common/images/inventories-constructed-inventory-details.png
+   :alt: Inventories constructed inventory details
```
Comment (gundalow) [inline docs/docsite/rst/userguide/inventories.rst]:
```suggestion
   :alt: Constructed inventory details
```

File: docs/docsite/rst/userguide/inventories.rst [hunk 3]
```diff
@@ -515,6 +527,7 @@ The type of inventory is identified at the top of the create form.
 |Inventories_create_new - create new inventory|
 
 .. |Inventories_create_new - create new inventory| image:: ../common/images/inventories-create-new-inventory.png
+   :alt: Inventories create new create new inventory
```
Comment (gundalow) [inline docs/docsite/rst/userguide/inventories.rst]:
```suggestion
   :alt: Create new create new inventory
```

File: docs/docsite/rst/userguide/inventories.rst [hunk 4]
```diff
@@ -528,6 +541,7 @@ The type of inventory is identified at the top of the create form.
 - **Instance Groups**: Click the |search| button to open a separate window. Choose the instance group(s) for this inventory to run on. If the list is extensive, use the search to narrow the options. You may select multiple instance groups and sort them in the order you want them ran.
 
 .. image:: ../common/images/select-instance-groups-modal.png
+   :alt: Select instance groups modal
```
Comment (gundalow) [inline docs/docsite/rst/userguide/inventories.rst]:
```suggestion
   :alt: Select instance groups dialogue
```

**Review by gundalow — CHANGES_REQUESTED** [review]
Thanks for this, just a couple of minor suggestions

File: docs/docsite/rst/userguide/inventories.rst [hunk 5]
```diff
@@ -170,10 +173,12 @@ To use ``ansible_facts`` to define the host filter when creating Smart Inventori
 1. In the *Create new smart inventory screen*, click the |search| button next to the **Smart host filter** field to open a pop-up window to filter hosts for this inventory.
 
 .. image:: ../common/images/inventories-smart-create-filter-highlighted.png
+   :alt: Inventories smart create filter highlighted
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:176]: Suggested rewording to sound more natural: "Create smart inventory window with Smart host filter highlighted"

File: docs/docsite/rst/userguide/inventories.rst [hunk 6]
```diff
@@ -170,10 +173,12 @@ To use ``ansible_facts`` to define the host filter when creating Smart Inventori
...
 
 2. In the search pop-up window, change the search criteria from **Name** to **Advanced** and select **ansible_facts** from the **Key** field.
 
 .. image:: ../common/images/inventories-smart-define-host-filter.png
+   :alt: Inventories smart define host filter
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:181]: Suggested wording: "Host filter with drop-down menu options"

File: docs/docsite/rst/userguide/inventories.rst [hunk 7]
```diff
@@ -186,24 +191,29 @@ If you wanted to add an ansible fact of
 In the search field, enter ``ansible_processor[]="GenuineIntel"`` (no extra spaces or ``__`` before the value) and press **[Enter]**.
 
 .. image:: ../common/images/inventories-smart-define-host-filter-facts.png
+   :alt: Inventories smart define host filter facts
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:194]: Suggested wording: "Example of an advanced search for host filter ansible facts"

File: docs/docsite/rst/userguide/inventories.rst [hunk 8]
```diff
@@ -186,24 +191,29 @@ If you wanted to add an ansible fact of
...
 
 The resulting search criteria for the specified ansible fact populates in the lower part of the window.
 
 .. image:: ../common/images/inventories-smart-define-host-filter-facts2.png
+   :alt: Second inventories smart define host filter facts
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:199]: Suggested wording: "Selected search criteria for host filter ansible facts listed below search field"

File: docs/docsite/rst/userguide/inventories.rst [hunk 9]
```diff
@@ -186,24 +191,29 @@ If you wanted to add an ansible fact of
...
 
 3. Click **Select** to add it to the **Smart host filter** field.
 
 .. image:: ../common/images/inventories-smart-create-filter-added.png
+   :alt: Inventories smart create filter added
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:204]: Suggested wording: "Specified search criteria for host filter ansible facts shown in the Smart host filter field of the Create new smart inventory window"

File: docs/docsite/rst/userguide/inventories.rst [hunk 10]
```diff
@@ -186,24 +191,29 @@ If you wanted to add an ansible fact of
...
 
 4. Click **Save** to save the new Smart Inventory.
 
 The Details tab of the new Smart Inventory opens and displays the specified ansible facts in the **Smart host filter** field.
 
 .. image:: ../common/images/inventories-smart-create-details.png
+   :alt: Inventories smart create details
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:211]: Suggested wording: "Details tab of the new Smart Inventory displaying the specified ansible facts in the Smart host filter field"

File: docs/docsite/rst/userguide/inventories.rst [hunk 11]
```diff
@@ -186,24 +191,29 @@ If you wanted to add an ansible fact of
...
 
 5. From the Details view, you can edit the **Smart host filter** field by clicking **Edit** and delete existing filter(s), clear all existing filters, or add new ones.
 
 .. image:: ../common/images/inventories-smart-define-host-filter-facts-group.png
+   :alt: Inventories smart define host filter facts group
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:216]: Suggested wording: "Specified search criteria consisting of host filter ansible facts and groups listed below search field"

File: docs/docsite/rst/userguide/inventories.rst [hunk 12]
```diff
@@ -246,6 +256,7 @@ The hosts inside the input inventory will fit one condition, the other condition
 neither, or both. This results in four hosts total for demonstration purposes.
 
 .. image:: ../common/images/inventories-constructed-inventory-venn.png
+   :alt: Inventories constructed inventory Venn diagram
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:259]: Suggested wording: "Venn diagram describing the input inventory content for a constructed inventory"

File: docs/docsite/rst/userguide/inventories.rst [hunk 13]
```diff
@@ -515,6 +527,7 @@ The type of inventory is identified at the top of the create form.
 |Inventories_create_new - create new inventory|
 
 .. |Inventories_create_new - create new inventory| image:: ../common/images/inventories-create-new-inventory.png
+   :alt: Create new create new inventory
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:530]: Suggested wording: "Create new inventory form"

File: docs/docsite/rst/userguide/inventories.rst [hunk 14]
```diff
@@ -555,10 +570,12 @@ The type of inventory is identified at the top of the create form.
   - **Source vars** for constructed inventories creates groups, specifically under the ``groups`` key of the data. It accepts Jinja2 template syntax, renders it for every host, makes a ``True``/``False`` evaluation, and includes the host in the group (from key of the entry) if the result is ``True``. This is particularly useful because you can paste that group name into the limit field to only include hosts in that group. See an example :ref:`here <constr_inv_example>`.
 
 .. |search| image:: ../common/images/search-button.png
+   :alt: Search button
 
 |Inventories_create_new_saved - create new inventory|
 
 .. |Inventories_create_new_saved - create new inventory| image:: ../common/images/inventories-create-new-saved-inventory.png
+   :alt: Create new inventory
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:578]: Suggested wording: "Example Create new inventory form filled out"

File: docs/docsite/rst/userguide/inventories.rst [hunk 15]
```diff
@@ -602,6 +619,7 @@ To create a new group for an inventory:
 |Inventories_manage_group_add|
 
 .. |Inventories_manage_group_add| image:: ../common/images/inventories-add-group-new.png
+   :alt: Inventores manage group add
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:622]: Suggested wording: "Create new group form for inventories"

File: docs/docsite/rst/userguide/inventories.rst [hunk 16]
```diff
@@ -634,12 +652,14 @@ The **Create Group** window closes and the newly created group displays as an en
 |Inventories add group subgroup|
 
 .. |Inventories add group subgroup| image:: ../common/images/inventories-add-group-subgroup-added.png
+   :alt: Inventories add group subgroup
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:655]: Suggested wording: "Related Groups tab of the Groups form for inventories"

File: docs/docsite/rst/userguide/inventories.rst [hunk 17]
```diff
@@ -634,12 +652,14 @@ The **Create Group** window closes and the newly created group displays as an en
...
 
 If you chose to add an existing group, available groups will appear in a separate selection window.
 
 |Inventories add group existing subgroup|
 
 .. |Inventories add group existing subgroup| image:: ../common/images/inventories-add-group-existing-subgroup.png
+   :alt: Inventories add group existing subgroup
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:662]: Suggested wording: "Existing group appearing in a separate selection window"

File: docs/docsite/rst/userguide/inventories.rst [hunk 18]
```diff
@@ -655,6 +675,7 @@ The list view displays all your inventory groups at once, or you can filter it t
 You may be able to delete a subgroup without concern for dependencies, since AWX will look for dependencies such as any child groups or hosts. If any exists, a confirmation dialog displays for you to choose whether to delete the root group and all of its subgroups and hosts; or promote the subgroup(s) so they become the top-level inventory group(s), along with their host(s).
 
 .. image:: ../common/images/inventories-groups-delete-root-with-children.png
+   :alt: Inventories group delete root with children
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:678]: Suggested wording: "Delete group confirmation dialog box with a prompt to select whether to delete all groups and hosts or promote child groups and hosts"

File: docs/docsite/rst/userguide/inventories.rst [hunk 19]
```diff
@@ -684,12 +706,14 @@ The **Create Host** window closes and the newly created host displays as an entr
 |Inventories add group host|
 
 .. |Inventories add group host| image:: ../common/images/inventories-add-group-host-added.png
+   :alt: Inventories add group host
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:709]: Suggested wording: "Hosts tab of the Groups window showing available hosts"

File: docs/docsite/rst/userguide/inventories.rst [hunk 20]
```diff
@@ -684,12 +706,14 @@ The **Create Host** window closes and the newly created host displays as an entr
...
 
 If you chose to add an existing host, available hosts will appear in a separate selection window.
 
 |Inventories add existing host|
 
 .. |Inventories add existing host| image:: ../common/images/inventories-add-existing-host.png
+   :alt: Inventories add existing host
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:716]: Suggested wording: "Existing host appearing in a separate selection window"

File: docs/docsite/rst/userguide/inventories.rst [hunk 21]
```diff
@@ -704,12 +728,14 @@ list of hosts.
 |Inventories add group host emphasized|
 
 .. |Inventories add group host emphasized| image:: ../common/images/inventories-add-group-host-added-emphasized.png
+   :alt: Inventories add group host emphasized
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:731]: Just host, not group host, despite the image name.

File: docs/docsite/rst/userguide/inventories.rst [hunk 22]
```diff
@@ -704,12 +728,14 @@ list of hosts.
...
 
 This opens the Details tab of the selected host.
 
 |Inventories add group host details|
 
 .. |Inventories add group host details| image:: ../common/images/inventories-add-group-host-details.png
+   :alt: Inventories add group host details
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:738]: Suggested wording: "Host details tab for the selected inventory"

File: docs/docsite/rst/userguide/inventories.rst [hunk 23]
```diff
@@ -718,6 +744,7 @@ This opens the Details tab of the selected host.
     Available groups appear in a separate selection window.
 
     .. image:: ../common/images/inventories-add-group-hosts-add-groups.png
+       :alt: Inventories add group hosts add groups
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:747]: Suggested wording: "Select Groups dialog showing two available groups"

File: docs/docsite/rst/userguide/inventories.rst [hunk 24]
```diff
@@ -726,6 +753,7 @@ This opens the Details tab of the selected host.
 8. If a host was used to run a job, you can view details about those jobs in the **Completed Jobs** tab of the host and click **Expanded** to view details about each job.
 
 .. image:: ../common/images/inventories-add-host-view-completed-jobs.png
+   :alt: Inventories add host view completed jobs
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:756]: Suggested wording: "Jobs tab showing list of completed jobs associated with the selected host"

File: docs/docsite/rst/userguide/inventories.rst [hunk 25]
```diff
@@ -750,6 +778,7 @@ This opens the Create Source window.
 |Inventories create source|
 
 .. |Inventories create source| image:: ../common/images/inventories-create-source.png
+   :alt: Inventories create source
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:781]: Suggested wording: "Create new source form for inventory source"

File: docs/docsite/rst/userguide/inventories.rst [hunk 26]
```diff
@@ -816,6 +845,7 @@ This opens the Create Source window.
     The **Notifications** tab is only present after you save the newly-created source.
 
     .. image:: ../common/images/inventories-create-source-with-notifications-tab.png
+       :alt: Inventories create source with notification tab
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:848]: Suggested wording: "Notification tab for the inventory source"

File: docs/docsite/rst/userguide/inventories.rst [hunk 27]
```diff
@@ -831,6 +861,7 @@ Once a source is defined, it displays as an entry in the list of sources associa
 |Inventories view sources|
 
 .. |Inventories view sources| image:: ../common/images/inventories-view-sources.png
+   :alt: Inventories view source
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:864]: Suggested wording: "Sources tab of the inventory showing one inventory source"

File: docs/docsite/rst/userguide/inventories.rst [hunk 28]
```diff
@@ -860,12 +891,14 @@ An inventory that is sourced from a project means that is uses the SCM type from
...
 
     .. image:: ../common/images/projects-create-scm-project-branch-override-checked.png
+       :alt: Project create scm project branch override checked
 
   - **Credential**: Optionally specify the credential to use for this source.
   - **Project**: Required. Pre-populates with a default project, otherwise, specify the project this inventory is using as its source. Click the |search| button to choose from a list of projects. If the list is extensive, use the search to narrow the options.
   - **Inventory File**: Required. Select an inventory file associated with the sourced project. If not already populated, you can type it into the text field within the drop down menu to filter the extraneous file types. In addition to a flat file inventory, you can point to a directory or an inventory script.
 
     .. image:: ../common/images/inventories-create-source-sourced-from-project-filter.png
+       :alt: Inventories create source sourced from project filter
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:901]: Suggested wording: "Inventory file field of the Sourced from a project inventory type"

File: docs/docsite/rst/userguide/inventories.rst [hunk 29]
```diff
@@ -1081,6 +1123,7 @@ Red Hat Ansible Automation Platform
 3. You can optionally specify the verbosity, host filter, enabled variable/value, and update options as described in the main procedure for :ref:`adding a source <ug_add_inv_common_fields>`.
 
   .. image:: ../common/images/inventories-create-source-rhaap-example.png
+     :alt: Inventories create source rhaap example
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:1126]: Spell out rhaap ("Red Hat Ansible Automation Platform")

File: docs/docsite/rst/userguide/inventories.rst [hunk 30]
```diff
@@ -1216,10 +1261,12 @@ The Run command window opens.
 |ad hoc-commands-inventory-run-command|
 
 .. |ad hoc-commands-inventory-run-command| image:: ../common/images/ad-hoc-commands-inventory-run-command.png
+   :alt: ad hoc commands inventory run command
 
 4. Click **Next** to choose the |ee| you want the ad-hoc command to be run against.
 
 .. image:: ../common/images/ad-hoc-commands-inventory-run-command-ee.png
+   :alt: ad hoc inventory run command ee
```
Comment (tvo318) [inline docs/docsite/rst/userguide/inventories.rst:1269]: Suggested wording: "Ad Hoc Run command dialog prompting for Execution Environments with two listed to choose from"

**Review by tvo318 — CHANGES_REQUESTED** [review]
Thank you for your contributions, @Dule-martins! 👍 I've made some minor wording suggestions so they are more descriptive when a screen reader reads to a user who has difficulty viewing the images.

**Comment by tvo318** [comment]
Closed this PR and re-opened new PR (#15394) with all the changes suggested.

**Pull request closed without merging**